src/Chainweb/Pact/Types.hs

```diff
@@ -286,6 +286,15 @@ moduleCacheKeys :: ModuleCache -> [ModuleName]
 moduleCacheKeys (ModuleCache a) = fst <$> LHM.toList a
 {-# INLINE moduleCacheKeys #-}
 
+-- this can't go in Chainweb.Version.Guards because it causes an import cycle
+-- it uses genesisHeight which is from BlockHeader which imports Guards
```

Comment:
Would it make sense to move `genesisHeight` to a different place instead?

Reply:
Probably would cause a cycle. I tried it during recordification.